C/casacore/build_tarballs.jl

```diff
@@ -0,0 +1,121 @@
+# Also exclude FreeBSD, lots of upstream bugs that seem not worth fixing ourself
+platforms = supported_platforms(experimental=true, exclude=(platform)-> Sys.iswindows(platform) || Sys.isfreebsd(platform))
+# Deal with the fact that we have std::string values, which causes issues across the gcc 4/5 boundary
+platforms = expand_cxxstring_abis(platforms)
+# expand gfortran versions as well
+platforms = expand_gfortran_versions(platforms)
+# Exclude all musl builds, upstream doesn't care and they use a few glibc-specific features
+filter!(p -> libc(p) != "musl", platforms)
+# Exclude powerpc64le-linux-gnu-libgfortran3, something is very wonky with that particular combination
+# Is it worth fixing?
+filter!(p -> !(arch(p) == "powerpc64le" && libgfortran_version(p) == v"3"), platforms)
+
+# The products that we will ensure are always built
+products = Product[
+    LibraryProduct("libcasa_casa", :libcasa_casa),
+    LibraryProduct("libcasa_coordinates", :libcasa_coordinates),
+    LibraryProduct("libcasa_derivedmscal", :libcasa_derivedmscal),
+    LibraryProduct("libcasa_fits", :libcasa_fits),
+    LibraryProduct("libcasa_images", :libcasa_images),
+    LibraryProduct("libcasa_lattices", :libcasa_lattices),
+    LibraryProduct("libcasa_meas", :libcasa_meas),
+    LibraryProduct("libcasa_measures", :libcasa_measures),
+    LibraryProduct("libcasa_mirlib", :libcasa_mirlib),
+    LibraryProduct("libcasa_msfits", :libcasa_msfits),
+    LibraryProduct("libcasa_ms", :libcasa_ms),
+    LibraryProduct("libcasa_scimath_f", :libcasa_scimath_f),
+    LibraryProduct("libcasa_scimath", :libcasa_scimath),
+    LibraryProduct("libcasa_tables", :libcasa_tables),
+    ExecutableProduct("findmeastable", :findmeastable),
+    ExecutableProduct("fits2table", :fits2table),
+    ExecutableProduct("image2fits", :image2fits),
+    ExecutableProduct("imagecalc", :imagecalc),
+    ExecutableProduct("imageregrid", :imageregrid),
+    ExecutableProduct("imageslice", :imageslice),
+    ExecutableProduct("lsmf", :lsmf),
+    ExecutableProduct("measuresdata", :measuresdata),
+    ExecutableProduct("measuresdata-update", :measuresdata_update),
+    ExecutableProduct("ms2uvfits", :ms2uvfits),
+    ExecutableProduct("msselect", :msselect),
+    ExecutableProduct("readms", :readms),
+    ExecutableProduct("showtableinfo", :showtableinfo),
+    ExecutableProduct("showtablelock", :showtablelock),
+    ExecutableProduct("tablefromascii", :tablefromascii),
+    ExecutableProduct("taql", :taql),
+    ExecutableProduct("tomf", :tomf),
+    ExecutableProduct("writems", :writems),
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="OpenBLAS32_jll", uuid="656ef2d0-ae68-5445-9ca0-591084a874a2"))
+    Dependency(PackageSpec(name="FFTW_jll", uuid="f5851436-0d7a-5f13-b9de-f02708fd171a"))
+    Dependency(PackageSpec(name="CFITSIO_jll", uuid="b3e40c51-02ae-5482-8a39-3ace5868dcf4"))
```

Comment:
I think it'd be better to specify `compat` for fftw,CFITSIO, WCS and readline: https://docs.binarybuilder.org/stable/reference/#BinaryBuilderBase.Dependency

Reply:
Compat to the minor versions should be good, yes?

Reply:
For the time being I'd use the major version. Upstream packages can break the ABI in a completely arbitrary way, or maybe even not break it. It's better to have some compat bounds we can easily fix later if necessary, but let's not be too restrictive